apps/aecore/lib/aecore/channel/channel_offchain_update.ex

```diff
@@ -0,0 +1,128 @@
+defmodule Aecore.Channel.ChannelOffchainUpdate do
```

Comment:
To follow existing patterns please rename to OffChain (capital C) and change filename accordingly (`git mv`)

Reply:
This applies also to other files.